generate_wrappers.py

```diff
@@ -361,6 +361,17 @@ def maybe_named(name):
     return '_ ' + name
   return name
 
+# Two cases based on if handle_var is TensorHandle or [TensorHandle]:
+# 1. Case TensorHandle: h -> Tensor(handle: h)
+# 2. Case [TensorHandle]: h -> h.map(Tensor.init)
+def convert_handle_to_tensor(handle_var, is_list):
+  if is_list:
+    return handle_var + '.map(Tensor.init)'
+  return 'Tensor(handle: ' + handle_var + ')'
+
+# If type_name is [TensorHandle<T>], it is a list type.
+def type_is_list(type_name):
```

Comment:
The implementation seems a bit fragile, maybe we could make this more robust by including in return_name_and_types whether this return value has a list type.

Reply:
Done.